core/domain/user_jobs_one_off_test.py

```diff
@@ -377,3 +385,443 @@ def test_adding_exploration_to_collection(self):
             user_b_subscriptions_model.activity_ids, [])
         self.assertEqual(
             user_b_subscriptions_model.collection_ids, [self.COLLECTION_ID_1])
+
+class UserImpactScoreOneOffJobTest(test_utils.GenericTestBase):
+    """ Tests the calculation of a user's impact score from the one-off
+    UserImpactCalculationOneOffJob.
+    """
+
+    EXP_ID_1 = 'exp_id_1'
+    EXP_ID_2 = 'exp_id_2'
+    USER_A_EMAIL = 'a@example.com'
+    USER_A_USERNAME = 'a'
+    # Constants imported from the oneoff job.
+    NUM_RATINGS_SCALER_CUTOFF = user_jobs_one_off.UserImpactCalculationOneOffJob.NUM_RATINGS_SCALER_CUTOFF
```

Comment:
Should I do this everywhere I reference user_jobs_one_off.UserImpactCalculationOneOffJob? So like, say self.impact_one_off_job instead of the long string of user_jobs_one_off.UserImpactCalculationOneOffJob? or would that look too strange?